Framework/Algorithms/test/RunCombinationHelperTest.h

```diff
@@ -40,6 +40,20 @@ class RunCombinationHelperTest : public CxxTest::TestSuite {
     m_testee.setReferenceProperties(m_reference);
   }
 
+  void testUnwraping_throws() {
+
+    MatrixWorkspace_sptr ws1 = create2DWorkspace(2, 3);
+    storeWS("ws1", ws1);
+
+    TS_ASSERT_THROWS_EQUALS(
+        m_testee.unWrapGroups(std::vector<std::string>{"ws1", "ws?"}),
+        const std::runtime_error &e, std::string(e.what()),
+        "Unable to find workspace type with name 'ws?': data service  search "
+        "object ws?");
+
+    removeWS("ws1");
+  }
+
```

Comment:
Could we also test that trying to conjoin invalid workspaces such as tables throws?

Reply:
That's true, the std::runtime error with message "The input ... is neither a WorkspaceGroup nor a MatrixWorkspace" should be tested instead.